service-front/assets/sass/patterns/_details.scss

```diff
@@ -9,3 +9,12 @@ details {
         margin-top: em(10, 20);
     }
 }
+
+// remove the arrow added by the govuk CSS :before selector
+// on summary elements inside a polyfilled <details>
+[data-polyfilled-details=true] .govuk-details__summary:before {
```

Comment:
Very picky but this could also be like this to make it more modular

`
[data-polyfilled-details=true] .govuk-details__summary {
    padding-left: 0 !important;
    
   &:before {
    display: none !important;
   }
}`

Reply:
Yeah, but I hate SASS

Reply:
I mean, I'm being flippant there, but I think Sass can make people lazy. Rather than thinking about connecting styles directly to markup, perhaps by marking elements with their role and associating CSS selectors with that, it can result in horribly nested selectors which are difficult to follow.

I would also argue it obfuscates findability. As the selector is split up across declarations (as here, where the :before is on an & symbol), you can't search directly for the `.govuk-details__summary:before` selector in your Sass files.

Finally, it pulls in a load of additional dependencies (Ruby) and needs to be compiled all the time during development, requiring a watcher process etc.

I have added the styles into the Sass file here, but only because that's how the project is set up. If I had worked on the project from the start, it would probably not have any Sass files in it at all!